config.toml

```diff
@@ -1,11 +1,12 @@
-baseURL = "https://pulumi.io/"
+baseURL = "https://pulumi.com/"
```

Comment:
Shouldn't this be `https://www.pulumi.com/` (with the `www`)?

Right now `https://pulumi.com` will redirect to `https://www.pulumi.com`. Personally, I wish it was the other way around and `https://pulumi.com` was the canonical URL. @chrsmith, do you remember why we decided to go with `www.pulumi.com` over `pulumi.com`? Should we revisit as part of this website merge effort? (I'm a little afraid if we switch, it'll somehow affect our SEO; but if we want to switch, for a more succinct URL, it'd probably be better to do it sooner than later).

Reply:
This is one of those things that feels a bit squishy to me, but if I had to make a call now, I'd probably suggest going with `www`, yeah. Other than aesthetics, there doesn't seem to be any downside to it, whereas with a bare domain, a handful of technical issues come into play (cookies, DNS options, etc.) that could present complications for us down the road. So I'm okay rolling with www if you all are.